src/test/resources/lang/exp/xhtml/lang-areg.html

```diff
@@ -47,21 +47,21 @@ <h1 class="title topictitle1" id="ariaid-title1">Topic elements with generated t
 
 <p class="p">Xref to the concept in this file: <a class="xref" href="lang-common.html#concept">Concept elements with generated text</a>. Now do it again, <a class="xref" href="lang-common.html#concept">with link text</a>.</p>
 
-<p class="p">Xref to the fig in this topic: <a class="xref" href="#topic__d7e166">شكل 1</a>. Now do it again, <a class="xref" href="#topic__d7e166">with link text</a>.</p>
+<p class="p">Xref to the fig in this topic: <a class="xref" href="#topic__d22e167">#topic__d22e167</a>. Now do it again, <a class="xref" href="#topic__d22e167">with link text</a>.</p>
```

Comment:
Multiple files in the  `lang` test change because it uses conref to build up the source documents.